cmd/container-use/agent/configure_claude.go

```diff
@@ -77,7 +75,22 @@ func (c *ConfigureClaude) editMcpConfig() error {
 func (c *ConfigureClaude) updateSettingsLocal(config ClaudeSettingsLocal) ([]byte, error) {
 	// Initialize permissions map if nil
 	if config.Permissions == nil {
-		config.Permissions = &ClaudePermissions{Allow: []string{}}
+		config.Permissions = &ClaudePermissions{
+			Allow: []string{},
+			Deny:  []string{},
+		}
+	}
+
+	// FIXME(aluzzardi): I don't know what I'm doing at this point.
+	// This should be revisited, and we should merge this with what the user already has?
+	config.Permissions.Deny = []string{
+		"LS",
+		"Read",
+		"Edit",
+		"MultiEdit",
+		"Write",
+		"Bash",
+		"Search",
 	}
```

Comment:
this behaves correctly for me, but i think we should also exclude Glob, Grep, NotebookRead, and NotebookEdit

Reply:
Done